dump/mapping/bufr_adpupa_prepbufr_mapping.yaml

```diff
@@ -0,0 +1,371 @@
+      units: "m s-1"
+
+    # ObsValue: windNorthward
+    - name: "ObsValue/windNorthward"
+      source: variables/windNorthward
+      longName: "Northward Wind"
+      units: "m s-1"
+
+    # ObsValue: specificHumidity
+    - name: "ObsValue/specificHumidity"
+      source: variables/specificHumidity
+      longName: "Specific Humidity"
+      units: "kg kg-1"
+
+    # QualityMarker: pressure
+    - name: "QualityMarker/pressure"
+      source: variables/pressureQualityMarker
+      longName: "Pressure Quality Marker"
+
+    # QualityMarker: height
+    - name: "QualityMarker/height"
+      source: variables/heightQualityMarker
+      longName: "Height Quality Marker"
+
+    # QualityMarker: stationPressure
+    - name: "QualityMarker/stationPressure"
+      source: variables/stationPressureQualityMarker
+      longName: "Station Pressure Quality Marker"
+
+    # QualityMarker: airTemperature
+    - name: "QualityMarker/airTemperature"
+      source: variables/airTemperatureQualityMarker
+      longName: "Temperature Quality Marker"
+
+    # QualityMarker: virtualTemperature
+    - name: "QualityMarker/virtualTemperature"
+      source: variables/virtualTemperatureQualityMarker
+      longName: "Virtual Temperature Quality Marker"
+
+    # QualityMarker: windEastward
+    - name: "QualityMarker/windEastward"
+      source: variables/windQualityMarker
+      longName: "Eastward Wind Quality Marker"
+
+    # QualityMarker: windNorthward
+    - name: "QualityMarker/windNorthward"
+      source: variables/windQualityMarker
+      longName: "Northward Wind Quality Marker"
+
+    # QualityMarker: specificHumidity
+    - name: "QualityMarker/specificHumidity"
+      source: variables/specificHumidityQualityMarker
+      longName: "Specific Humidity Quality Marker"
+
+    # ObsError: pressure 
+    - name: "ObsError/pressure"
+      source: variables/pressureError
+      longName: "Pressure Error"
+      units: "Pa"
+
+    # ObsError: stationPressure
+    - name: "ObsError/stationPressure"
+      source: variables/stationPressureError
+      longName: "Station Pressure Error"
+      units: "Pa"
+
+    # ObsError: airTemperature
+    - name: "ObsError/airTemperature"
+      source: variables/airTemperatureError
+      longName: "Temperature Error"
+      units: "K"
+
+    # ObsError: virtualTemperature
+    - name: "ObsError/virtualTemperature"
+      source: variables/virtualTemperatureError
+      longName: "Virtual Temperature Error"
+      units: "K"
+
+    # ObsError: windEastward
+    - name: "ObsError/windEastward"
+      source: variables/windError
+      longName: "East Wind Error"
+      units: "m s-1"
+
+    # ObsError: windNorthward
+    - name: "ObsError/windNorthward"
+      source: variables/windError
+      longName: "North Wind Error"
+      units: "m s-1"
+
+    # ObsError: specificHumidity
+    - name: "ObsError/specificHumidity"
+      source: variables/specificHumidityError
+      longName: "Specific Humidity Error"
```

Comment:
Does this have a unit?

Reply:
@nicholasesposito thanks, added unit for specificHumidity.